Stylesheets/product.css

```diff
@@ -0,0 +1,63 @@
+body{
+    background-color: var(--st-white);
+}
+
+.little-margin-bottom{
+    margin-bottom: 0.5rem;
+}
+
+.product-status{
+    display: inline-block;
+   margin: 0 1rem 2rem 0.5rem;
+}
+
+.flex-spacebetween{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* card wishlist */
+.wishlist-badge{
+    width: 25px;
+    height: 23px;
+    background-color: var(--st-white);
+    border-radius: 50%;
+    position: absolute;
+    top:10px;
+    right:7px;
+    text-align: center;
+    color:var(--st-red)
+}
+
+.pdt-page-content{
+    display: flex;
+    justify-content: stretch;
+    
+}
+
+.side-bar{
+    width: 100%;
+    margin: 0rem 2rem;
+}
+
+.price-range{
+    display: flex;
```

Comment:
using utility here would be easier....as i can see you already used some flex utilities in your code